charts/prefect-agent/templates/_helpers.tpl

```diff
@@ -14,9 +14,9 @@ Create the name of the service account to use
     Define API URL for workspace or for
 */}}
 {{- define "agent.apiUrl" -}}
-{{- if and .Values.agent.config.accountId .Values.agent.config.workspaceId }}
-{{- printf "%s/api/accounts/%s/workspaces/%s" .Values.agent.config.apiUrl .Values.agent.config.accountId .Values.agent.config.workspaceId | quote }}
+{{- if eq .Values.agent.apiConfig "cloud" }}
+{{- printf "%s/accounts/%s/workspaces/%s" .Values.agent.cloudApiConfig.cloudUrl .Values.agent.cloudApiConfig.accountId .Values.agent.cloudApiConfig.workspaceId | quote }}
```

Comment:
I'm wondering if we could handle this instead by showing the API URL in the UI somewhere (we could have a page that explains to people how to install an agent for a given work queue, with an example command to run in the case of the Helm chart)

so maybe orion/cloud don't really need to be different after all?

Reply:
I don't think the Helm chart is mainstream enough to target from the UI. The core library already templates/generates the API URL when users do `prefect cloud login` and we've established that they should not be responsible for creating the path to their workspace. I think it's reasonable for the chart to do so as well 🤷‍♀️